src/wallet/rpc/wallet.cpp

```diff
@@ -40,7 +40,6 @@ static RPCHelpMan getwalletinfo()
                     {
                         {
                         {RPCResult::Type::STR, "walletname", "the wallet name"},
-                        {RPCResult::Type::NUM, "walletversion", "the wallet version"},
```

Comment:
I'd keep it for backwards compatibility.
`{RPCResult::Type::NUM, "walletversion", "(DEPRECATED) only related to unsupported legacy wallet, returns the latest version 169900 for backwards compatibility"},`

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/32977/commits/fc4d563edb5feab273e5770f1b369cb6fe8a0c8c